server/app.ts

```diff
@@ -60,7 +60,7 @@ const getCourseData = async function (req: Request) {
 new MathigonStudioApp()
   .get('/health', (req, res) => res.status(200).send('ok')) // Server Health Checks
   .secure()
-  .setup({ sessionSecret: 'project-platypus-beta' })
+  .setup({ sessionSecret: 'project-platypus-beta', csrfBlocklist: ['/profile/accept-policies'] })
```

Comment:
Apart from this that I'm not 100% confident with this approach I'm fine with the rest of the things. I can open an issue to investigate it a bit more but by now I think we can go with it.

Reply:
Thanks @techtolentino I will edit it with more information.